code/obj/machinery/door/feather.dm

```diff
@@ -13,6 +13,8 @@
 /obj/machinery/door/feather/New()
 	..()
 	setMaterial("gnesis")
+	var/datum/component/C = src.GetComponent(/datum/component/mechanics_holder)
+	C.RemoveComponent()
```

Comment:
```suggestion
	C?.RemoveComponent()
```
technically nullable in case this component ever disappears from doors in the future